pennylane/ops/channel.py

```diff
@@ -238,10 +274,24 @@ class DepolarizingChannel(Channel):
     def num_params(self):
         return 1
 
-    @classmethod
-    def _kraus_matrices(cls, *params):
-        p = params[0]
+    @staticmethod
+    def compute_kraus_matrices(p):  # pylint:disable=arguments-differ
+        r"""Kraus matrices representing the DepolarizingChannel channel.
```

Comment:
```suggestion
        r"""Kraus matrices representing the depolarizing channel.
```
reads a bit better? Might also apply to other methods above and below